ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/TxStatsMBean.java

```diff
@@ -53,6 +53,13 @@
     long getNumberOfCommittedTransactions();
 
     /**
+     * Returns the average time it is taking to commit a transaction.
+     * Note that a small number of stuck transactions can skew the overall average.
+     * @return the average time, in nanoseconds, it has taken to commit a transaction.
+     */
+    long getAverageCommitTime();
```

Comment:
Hi Mike, as per the comment on the forum I don't think this Javadoc definitively clarifies it is the timing of the commit() call not the time from begin() to commit() completing. I will reopen the issue for that aspect to be covered